motion_state_machine:
  Startup: [Controllable, Softoff, GettingUp, Record, PenaltyAnimationIn]
  Softoff: [Controllable, ShutDown, Record]
  Record: [Controllable]
  PenaltyAnimationIn: [Penalty]
  Penalty: [PenaltyAnimationOut]
  PenaltyAnimationOut: [Controllable]
  GettingUp: [Falling, Fallen, GettingUpSecond]
  GettingUpSecond: [Falling, Fallen, Controllable]
  Controllable: [ShutDownAnimation, Record, PenaltyAnimationIn, Softoff, Falling, Fallen,
                AnimationRunning, Walking]
  Falling: [Fallen, Controllable]
  Fallen: [GettingUp]
  Walking: [ShutDownAnimation, Record, PenaltyAnimationIn, Softoff, Falling, Fallen,
           WalkingStopping, Controllable]
  WalkingStopping: [Controllable]
  AnimationRunning: [Controllable]
  ShutDownAnimation: [ShutDown]